package.json

```diff
@@ -1,29 +1,55 @@
 {
-  "name": "gpt-3-encoder",
-  "version": "1.1.3",
-  "description": "Javascript BPE Encoder Decoder for GPT-2 / GPT-3",
+  "name": "@syonfox/gpt-3-encoder",
```

Comment:
#30 @rossanodr  

I believe when you npm install the "package manager" uses the name to specify the package from here.  

SO if you install the @syonfox version you need to also used @syonfox/gpt-3-encoder